openapi.yaml

```diff
@@ -6995,6 +6995,90 @@ paths:
       - lang: CLI
         source: >
           linode-cli longview delete 789
+  /longview/plan:
+    x-linode-cli-command: longview
+    get:
+      tags:
+      - Longview
+      summary: View Longview Plan
+      description: >
+        Returns the Longview subscription plan for this user. You can send a request to the
+        [View Longview Subscription](/api/v4/longview-subscriptions-subscription-id) endpoint for details on each particular subscription plan.
+      operationId: getLongviewPlan
+      x-linode-cli-action: plan-view
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - longview:read_only
+      responses:
+        '200':
+          description: The requested Longview Subscription.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/LongviewPlan'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+              https://api.linode.com/v4/longview/plan
+      - lang: CLI
+        source: >
+          linode-cli longview plan-view
+    put:
+      tags:
+      - Longview
+      summary: Update Longview Plan
+      description: >
+        Updates the Longview subscription plan for this user. You can send a request to the
+        [View Longview Subscription](/api/v4/longview-subscriptions-subscription-id) endpoint for details on each particular subscription plan.
+      operationId: updateLongviewPlan
+      x-linode-cli-action: plan-update
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - longview:read_write
```

Comment:
Account:read_write is required here, same as the account settings location for changing the subscription plan.